src/uuid/empty.cr

```diff
@@ -0,0 +1,14 @@
+struct UUID
+  # Empty UUID.
+  @@empty_bytes = StaticArray(UInt8, 16).new { 0_u8 }
```

Comment:
It should be a constant, along with some other namespaces:

```crystal
EMPTY = UUID.new(StaticArray(UInt8, 16).new(0_u8))
DNS   = UUID.new("6ba7b810-9dad-11d1-80b4-00c04fd430c8")
URL   = UUID.new("6ba7b811-9dad-11d1-80b4-00c04fd430c8")
OID   = UUID.new("6ba7b812-9dad-11d1-80b4-00c04fd430c8")
X500  = UUID.new("6ba7b814-9dad-11d1-80b4-00c04fd430c8")
```

I don't think we'd need the `#empty` method anymore —since UUID is a struct it will always be copied.